src/App.tsx

```diff
@@ -1,14 +1,17 @@
-import React from "react";
 import "./App.css";
 
 import { taskList } from "./taskList";
 
 function TaskShow() {
   return (
     <div>
```

Comment:
this outer div is the one that can have the `padding` of `24px` so you don't have to add padding to `title` and `container`